react/src/components/organisms/TabContainer/tab.js

```diff
@@ -1,53 +1,117 @@
-import React from 'react';
+import React, { useContext, useCallback } from 'react';
 import PropTypes from 'prop-types';
-import shortid from 'shortid';
 import classNames from 'classnames';
 import { TabContext } from './context';
 
-class Tab extends React.Component {
-  constructor(props) {
-    super(props);
-    this.tabIdent = shortid.generate();
-    this.defaultSet = false;
-  }
-
-  render() {
-    return(
-      <TabContext.Consumer>
-        {(context) => {
-          const { activeTab, setActiveTab } = context;
-          const active = (activeTab === this.tabIdent);
-          const tabClasses = classNames({
-            'ma__tab-title': true,
-            'ma__tab-title--active': active
-          });
-          if (!this.defaultSet && this.props.default === true) {
-            // eslint-disable-next-line react/prop-types
-            setActiveTab(this.tabIdent, this.props.children);
-            this.defaultSet = true;
+const Tab = React.forwardRef((props, ref) => {
+  const context = useContext(TabContext);
+  const { tabIdent, active } = props;
+  const handleKeyDown = (e) => {
+    if (e.key === 'ArrowDown') {
+      const body = document.getElementById(context.tabContainerBodyId);
+      body.setAttribute('tabindex', '0');
+      e.currentTarget.setAttribute('tabindex', '-1');
```

Comment:
I think you should get rid of this line, per @clairesunstudio comment in slack, this can let you tab in and out of the nested tabs/body
```suggestion
```